libcaf_openssl/src/openssl/session.cpp

```diff
@@ -243,13 +241,12 @@ SSL_CTX* session::create_ssl_context() {
     SSL_CTX_set1_groups_list(ctx, "P-384");
 #  endif /* OPENSSL_VERSION_NUMBER < 0x10101000L */
 #endif
-#ifdef CAF_SSL_HAS_SECURITY_LEVEL
```

Comment:
I think this is not used anywhere now and could be removed:
```
% rg CAF_SSL_HAS_SECURITY_LEVEL
libcaf_openssl/caf/openssl/session.hpp
21:#  define CAF_SSL_HAS_SECURITY_LEVEL
```

Reply:
Good catch! Removed. 👍